README.md

```diff
@@ -21,70 +21,120 @@ production-grade software switch like [Open
 vSwitch](https://www.openvswitch.org/). For more information about the
 performance of bmv2, refer to this [document](docs/performance.md).
 
-## Dependencies
+# Getting started
 
-The following packages are required:
+## Installing packaged versions of bmv2
 
-### Ubuntu 20.04
+bmv2 has package support for several [Ubuntu](#ubuntu) and [Debian](#debian) distributions.
 
-```console
-sudo apt-get install -y automake cmake libgmp-dev \
-    libpcap-dev libboost-dev libboost-test-dev libboost-program-options-dev \
-    libboost-system-dev libboost-filesystem-dev libboost-thread-dev \
-    libevent-dev libtool flex bison pkg-config g++ libssl-dev
+### Ubuntu
+
+A bmv2 package is available in the following repositories for Ubuntu 20.04 and newer.
+
+```bash
+. /etc/os-release
+echo "deb http://download.opensuse.org/repositories/home:/p4lang/xUbuntu_${VERSION_ID}/ /" | sudo tee /etc/apt/sources.list.d/home:p4lang.list
+curl -fsSL "https://download.opensuse.org/repositories/home:p4lang/xUbuntu_${VERSION_ID}/Release.key" | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/home_p4lang.gpg > /dev/null
+sudo apt update
+sudo apt install p4lang-bmv2
+```
+
+### Debian
+
+For Debian 11 (Bullseye) it can be installed as follows:
+
+```bash
+echo 'deb http://download.opensuse.org/repositories/home:/p4lang/Debian_11/ /' | sudo tee /etc/apt/sources.list.d/home:p4lang.list
+curl -fsSL https://download.opensuse.org/repositories/home:p4lang/Debian_11/Release.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/home_p4lang.gpg > /dev/null
+sudo apt update
+sudo apt install p4lang-bmv2
 ```
 
-You also need to install the following from source. Feel free to use the
-install scripts under `ci/`.
+If you cannot use a repository to install bmv2, you can download the `.deb` file
+for your release and install it manually. You need to download a new file each
+time you want to upgrade bmv2.
 
-- [thrift 0.11.0](https://github.com/apache/thrift/releases/tag/0.11.0) or later
-  (tested up to 0.13)
-- [nanomsg 1.0.0](https://github.com/nanomsg/nanomsg/releases/tag/1.0.0) or
-  later
+1. Go to https://build.opensuse.org/package/show/home:p4lang/p4lang-bmv2, click on
+"Download package" and choose your operating system version.
 
-### Fedora
+2. Install bmv2 using the command below with the corresponding path where the package has been downloaded.
 
-```console
-sudo dnf install -y automake cmake gmp-devel libpcap-devel \
-    boost-devel boost-system boost-thread boost-filesystem boost-test \
-    libevent-devel libtool flex bison pkg-config g++ openssl-devel \
-    thrift-devel nanomsg-devel
+```bash
+sudo dpkg -i /path/to/package.deb
 ```
 
-To use the CLI, you will need to install the
-[nnpy](https://github.com/nanomsg/nnpy) Python package. Feel free to use
-`ci/install-nnpy.sh`.
+## Installing bmv2 from source
```

Comment:
LGTM. Minor thing. Can we make `Installing packaged versions of bmv2`  `Installing bmv2 from source` subsections of the same `Installing bmv2` header?

Reply:
Thanks! I've updated the pull request.